src/index.ts

```diff
@@ -146,6 +149,7 @@ export {
   ExpectedOneOfValuesError,
   ExpectedValueOfTypeError,
   FunctionPlugin,
+  FunctionArgumentType,
```

Comment:
Is it a bug or a feature? Should it be described in changelog?

Reply:
Do you mean if the lack of export was a bug or a feature? I think it's a new feature that should be listed in the changelog.

Reply:
Ok, I described it in the changelog